recipes/n2v/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "n2v" %}
+{% set version = "0.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/n2v-{{ version }}.tar.gz
+  sha256: 0be1184438d529ea6629fdbe800bcd71f5baadb16f0c5765a859870db24a7a31
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - csbdeep >=0.6.0  # ,<0.7.0
+    - imagecodecs >=2020.2.18
+    - keras >=2.1.1  # ,<2.4.0
```

Comment:
What is the reason for the commented pins?

Reply:
Updated to create the recipe for a newer version!